Makefile

```diff
@@ -119,11 +119,13 @@ build_docker_dev:
 ############################################ Deps ############################################
 
 install_deps_linux:
-	PROTOC_ZIP=protoc-3.14.0-linux-x86_64.zip
-	curl -OL https://github.com/protocolbuffers/protobuf/releases/download/v3.14.0/$PROTOC_ZIP
+	curl -OL https://github.com/protocolbuffers/protobuf/releases/download/v3.17.0/protoc-3.17.0-linux-x86_64.zip
 	sudo unzip -o $PROTOC_ZIP -d /usr/local bin/protoc
 	sudo unzip -o $PROTOC_ZIP -d /usr/local 'include/*'
 	rm -f $PROTOC_ZIP
 
 install_deps_darwin:
-	brew install protobuf
\ No newline at end of file
+	curl -OL https://github.com/protocolbuffers/protobuf/releases/download/v3.17.0/protoc-3.17.0-osx-x86_64.zip
```

Comment:
there's no way to get the version into a variable so that we don't end up with these getting out of date the next time it gets updated?

Reply:
Done!